model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -327,6 +336,15 @@ message ProcessBundleResponse {
   // payload field with the bytes specified here.
   map<string, bytes> monitoring_data = 5;
 
+  // (optional) Output elements of the processed bundle. Either all or
+  // none of the bundle elements should be included in the ProcessBundleResponse.
+  // This embedding is to achieve better efficiency for bundles that only
+  // contain small amounts of data. his field can be set only if the runner
+  // declares that it supports the CONTROL_RESPONSE_ELEMENTS_EMBEDDING
```

Comment:
```suggestion
  // declares that it supports the beam:protocol:control_response_elements_embedding:v1
```

Note that the ENUM can change but its contents of string can't so best to use it within the comment.

Reply:
Done.